daemon/algod/api/algod.oas3.yml

```diff
@@ -1934,6 +1934,23 @@
         ],
         "type": "object"
       },
+      "ScratchChange": {
+        "description": "The write operation into a scratch slot.",
```

Comment:
```suggestion
        "description": "A write operation into a scratch slot.",
```

Reply:
this is generated from oas2 file, see [oas2 model description change](https://github.com/algorand/go-algorand/pull/5563/commits/17eed397fdf704e1bc809246231213481bc769bc)